WapTenantPublicAPI.psm1

```diff
@@ -2675,6 +2814,22 @@ function New-WAPVM {
         [ValidateNotNullOrEmpty()]
         [String] $Name,
 
+        [Parameter()]
+        [ValidateNotNullOrEmpty()]
+        [String] $ComputerName = $Name,
+
+        [Parameter()]
+        [ValidateNotNullOrEmpty()]
+        $HWProfile,
+
+        [Parameter()]
+        [ValidateNotNullOrEmpty()]
+        [int32] $CPU,
```

Comment:
Cannot be a negative number so please use uint. No need for ValidateNotNullOrEmtpy